content/docs/1.9.0/concepts.md

```diff
@@ -64,20 +64,20 @@ The Engine and replicas are orchestrated using Kubernetes.
 In the figure below,
 
 - There are three instances with Longhorn volumes.
-- Each volume has a dedicated controller, which is called the Longhorn Engine and runs as a Linux process.
-- Each Longhorn volume has two replicas, and each replica is a Linux process.
+- Each volume has a dedicated controller, called the Longhorn Engine. For V1 volumes, the engine runs as a Linux process, while for V2 volumes, it operates as an SPDK logical volume.
```

Comment:
I thought the SPDK logical volume should represent a replica when compared with V1.

Should we say SPDK Raid1 bdev for the v2 engine instead?

Reply:
Ah, yes, typo. Should be Raid1 bdev

Reply:
Updated. Thank you.